src/pudl/package_data/phmsagas/column_maps/yearly_inspections_and_assessments.csv

```diff
@@ -0,0 +1,103 @@
+interstate_or_intrastate ,,,,,,,,,,,,,,,,,,,,,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra,inter_intra
+report_state,,,,,,,,,,,,,,,,,,,,,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name,state_name
+inspected_corrosion_or_metal_loss_tools_miles,,,,,,,,,,,,,,,,,,,,,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a,partf1a
+inspected_dent_or_deformation_tools_miles,,,,,,,,,,,,,,,,,,,,,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b,partf1b
+inspected_crack_or_long_seam_defect_detection_tools_miles,,,,,,,,,,,,,,,,,,,,,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c,partf1c
+inspected_other_tools_miles,,,,,,,,,,,,,,,,,,,,,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d,partf1d
+other_tools_description,,,,,,,,,,,,,,,,,,,,,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1doth,partf1d1,partf1d1
+inspected_in_line_total_miles,,,,,,,,,,,,,,,,,,,,,partf1e,partf1e,partf1e,partf1e,partf1e,partf1e,partf1e,partf1e,partf1e,partf1e,partf1e,partf1tot,partf1tot
+anomalies_excavated_operator_criteria,,,,,,,,,,,,,,,,,,,,,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a,partf2a
+anomalies_repaired_following_in_line_inspection,,,,,,,,,,,,,,,,,,,,,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b,partf2b
+conditions_repaired_hca_segment_in_line,,,,,,,,,,,,,,,,,,,,,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c,partf2c
+conditions_repaired_immediate_repair_conditions_hca_segment_in_line,,,,,,,,,,,,,,,,,,,,,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1,partf2c1
+conditions_repaired_one_year_conditions_hca_segment_in_line,,,,,,,,,,,,,,,,,,,,,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2,partf2c2
+conditions_repaired_monitored_conditions_hca_segment_in_line,,,,,,,,,,,,,,,,,,,,,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3,partf2c3
+conditions_repaired_other_scheduled_conditions_hca_segment_in_line,,,,,,,,,,,,,,,,,,,,,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4,partf2c4
+conditions_repaired_192.710_segment_in_line,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,partf2d,partf2d
```

Comment:
I'd recommned `conditions_repaired_192_710_segment_in_line` instead of the `192.710` bc the `.`could junk up the df.column.name

Reply:
Ah I meant to do this and clearly missed it, will do.